vnext/Shared/DevServerHelper.h

```diff
@@ -33,15 +33,23 @@ class DevServerHelper {
       const std::string &platform,
       bool dev,
       bool hot,
-      bool inlineSourceMap) {
+      bool inlineSourceMap,
+      const uint32_t hermesBytecodeVersion) {
```

Comment:
>hermesBytecodeVersion [](http://example.com/codeflow?start=21&length=21)

(nit) So this is a general function, for all engines, right? Could we generalize it instead of making it hermes specific with hermesBytecodeVersion argument?

Reply:
Nikolai .. THanks for the suggestion ... But, I don't see any clean way to generalize it .. Renaming the arg to something more generic (say byteCodeVersion) might help, but i think keeping 'hermes' in the arg will make it more readable .. I'd prefer keeping it as is .. :(